### PR TITLE
Mvp/router header component

### PR DESCRIPTION
**Pull request by aaayumi**
https://waffle.io/chingu-voyage4/Bears-Team-16/cards/5a98600875b5a300257f8a6e

- React Router 
- Header component 

[in progress]
need to add tests

### CODE REVIEW

File: client/src/components/Home.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { gql } from "apollo-boost";
+import { graphql } from "react-apollo";
+
+const fetchRecipes = gql`
```
Comment (thinktwice13) [inline client/src/components/Home.js:5]: Added graphql integration and a sample query. Will work when it is merged into `development`

**Review by thinktwice13 — CHANGES_REQUESTED** [review]
Having both `package-lock.json` and `yarn.lock` will cause some issues later in deployment. 
Aside from that, all good :+1:

**Review by thinktwice13 — DISMISSED** [review]

**Comment by thomaslombart** [comment]
So what should we keep ? `package-lock` or `yarn.lock` ?

**Comment by thinktwice13** [comment]
Correction: GraphQL will work once both this and `feature/api-methods` are merged into development

@thomlom I don't like it, but I think `npm` is still more standard. But it won't be a problem for a while.

**Comment by aaayumi** [comment]
So I removed `yarn.lock`.

**Review by thomaslombart — COMMENTED** [review]
Looks good to me!

**Review by thomaslombart — DISMISSED** [review]

**Review by thomaslombart — APPROVED** [review]
Still looks good to me 🙂

**Review by fredhawk — APPROVED** [review]
Looks good to me. 👍

**Pull request merged**